resources/views/admin/groups/edit.blade.php

```diff
@@ -67,6 +67,30 @@
                             'v-bind:class' => '{\'form-control\':true, \'is-invalid\':errors.status}']) !!}
                             <div class="invalid-feedback" role="alert" v-if="errors.status">@{{errors.status[0]}}</div>
                         </div>
+
+                        @if ($settings['2fa_enabled'])
+                            <div class="form-group mt-3">
+                                <div class="custom-control custom-switch">
+                                    {!! Form::checkbox('enabled_2fa', 1, null, [
+                                        'id' => 'enabled_2fa',
+                                        'class' => 'custom-control-input',
+                                        'v-model' => 'formData.enabled_2fa',
+                                    ]) !!}
+
+                                    {!! Form::label('enabled_2fa', __('Two Factor Authentication'), [
+                                        'class' => 'custom-control-label',
+                                        'for' => 'enabled_2fa',
+                                    ]) !!}
+                                </div>
+
+                                <ul v-show="formData.enabled_2fa" v-cloak>
```

Comment:
Consider to remove the list of methods since it is not present in the mockups

![image](https://github.com/ProcessMaker/processmaker/assets/8028650/68cf247c-1ddb-47cc-9636-b5d4eaa45f4b)